starboard/build/config/BUILD.gn

```diff
@@ -126,12 +126,6 @@ config("starboard") {
       "COBALT",  # TODO: See if this can be replaced by STARBOARD macro.
     ]
 
-    if (gl_type == "none") {
```

Comment:
Should we add and assert that is it is not "none"?

Reply:
Added one, but in `starboard/egl_and_gles/BUILD.gn`, it fits better there.